app/assets/stylesheets/landing.scss

```diff
@@ -0,0 +1,178 @@
+nav {
+  border-radius: 3%;
+  height: 5em;
+}
+
+.medium {
+  height: 45em;
+  width: 100%;
+  background: url("http://i.imgur.com/CShEgXJ.jpg") no-repeat;
+  background-size: cover;
+  display: block;
+}
+
+.row-break {
+  height: 4em;
+  width: 100%;
+}
+
+h5.cat-heading::before, h5.cat-heading::after {
```

Comment:
Avoid qualifying class selectors with an element.<br>Each selector in a comma sequence should be on its own single line